src/components/Shared/UI/DTECheckList/DTECheckList.tsx

```diff
@@ -65,7 +65,7 @@ const DTECheckList = memo(({ id, name, label, hint, error, values, onValueChange
       disabled={item.disabled}
       conditional={item.conditional}
       onClick={(e: any) => handleOnChange(e)}
-      onKeyPress={(e: any) => handleOnChange(e)}
+      // onKeyPress={(e: any) => handleOnChange(e)}
```

Comment:
@tanishawPA why has this been commented out?

Reply:
The line was greyed out and said it was a deprecated function. I can go back and just remove the line? Wasn't super sure!

Reply:
@tanishawPA I have had a look and it's not being used in the application.  Just be sure if you are removing/commenting anything out you have a reason and understand why.  It looks like it is fine to remove so feel free to delete but if future make sure you understand the "why".  

Nothing should make it to github if you don't understand, as who knows what you could be affecting!